go.mod

```diff
@@ -91,4 +91,13 @@ require (
 	sigs.k8s.io/yaml v1.2.0 // indirect
 )
 
+require (
+	github.com/go-test/deep v1.0.7 // indirect
+	github.com/open-cluster-management/hub-of-hubs-data-types v0.1.1-0.20211025070346-60d0cacb5152 // indirect
+	github.com/open-cluster-management/multicloud-operators-channel v1.0.1-0.20201120143200-e505a259de45
+	github.com/open-cluster-management/multicloud-operators-subscription v1.2.2-2-20201130-59f96
+	golang.org/x/tools v0.1.5 // indirect
+	sigs.k8s.io/application v0.8.3
```

Comment:
which application version is used in ACM v2.4?

Reply:
checked again its the correct version